index.js

```diff
@@ -6,10 +6,12 @@ function createFunction(parameters, defaultType) {
     if (!isFunctionDefinition(parameters)) {
         fun = function() { return parameters; };
         fun.isFeatureConstant = true;
-        fun.isGlobalConstant = true;
+        fun.isZoomConstant = true;
 
     } else {
-        var property = parameters.property === undefined ? '$zoom' : parameters.property;
+        var zoomAndDataDependent = typeof parameters.stops[0][0] === 'object';
+        var dataDependent = zoomAndDataDependent || parameters.property !== undefined;
```

Comment:
I'm ok with talking about data-driven styling in general. This specific feature, however, I have been and will continue to refer to as "property functions."

We support "property functions", "zoom functions", and "property-and-zoom functions" all of which can be referred to as "functions."